hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -133,6 +133,10 @@
   private static final String MERGE_DATA_VALIDATION_CHECK_ENABLED = "hoodie.merge.data.validation.enabled";
   private static final String DEFAULT_MERGE_DATA_VALIDATION_CHECK_ENABLED = "false";
 
+  // Concats inserts to data files without merging
+  private static final String MERGE_ALLOW_DUPLICATE_INSERTS = "hoodie.merge.allow.duplicate.inserts";
```

Comment:
@vinothchandar : does the config name looks ok? somehow I feel it does not exactly convey the actual intent, but talks about the effect. For instance,
something like "hoodie.concat.inserts.without.merge.on.insert" or "hoodie.concat.records.on.insert" etc.

Reply:
first one is too long IMO. I think `hoodie.merge.allow.duplicates.on.insert` is a better name (I added the `.on`) given it explains to the user what the impact is. Whether we concat or prepend in implementation, thats not very useful for the end user